master

NickHeidinger opened the pull request:
fixed alignment, added form highlights, added slack link, header background needs work still

NickHeidinger committed:
fixed layout, icons, form titles, header backgound needs work

asikkenga commented on index.html:
```diff
@@ -214,11 +211,11 @@ <h4 ><span>Wynand</span> Badenhorst</h4>
                 <div class="exec_img">
-                    <img src="images/harman.jpg" alt="Harmon" />
+                    <img src="images/harman.jpg" alt="Harman" />
+                    <span class="title">Co-VP of Outreach</span>
```
Why did you move this up?  I only see the change on Harman's

NickHeidinger replied:
Moved up? I fixed a typeo in his name

asikkenga replied:
The span is moved inside the div. It belongs outside. Otherwise nice catch on the spelling

NickHeidinger replied:
oh i dont think it affects the page rendering

NickHeidinger replied:
fixed*

NickHeidinger committed:
fixed header pic (except iphone 6 sizing)
changed size of exec images
added mail.php
added projects.html
adjusted featurette sizings
added maxlength to contact form fields

NickHeidinger commented:
![projects-page](https://user-images.githubusercontent.com/17942416/27512313-edc11868-5901-11e7-8c64-faf4279c50a7.png)

NickHeidinger committed:
fixed span in harman div
